docs/docs/12_task_queues_emails/04_populate_rq_task_queue/end/migrations/env.py

```diff
@@ -81,8 +80,7 @@ def process_revision_directives(context, revision, directives):
             connection=connection,
             target_metadata=target_metadata,
             process_revision_directives=process_revision_directives,
-            compare_type=True,
-            **current_app.extensions['migrate'].configure_args
+                **current_app.extensions['migrate'].configure_args
```

Comment:
Same as the other one, is the indentation correct?

```suggestion
            **current_app.extensions['migrate'].configure_args
```